tests/System.Buffers.Primitives.Tests/BufferReferenceTesting.cs

```diff
@@ -237,5 +243,37 @@ static void BufferLifetime(Buffer<byte> buffer)
             //{
             //}
         }
+
+        static void BufferBasics(ReadOnlyBuffer<byte> buffer)
+        {
+            var span = buffer.Span;
+            Assert.Equal(buffer.Length, span.Length);
+            Assert.True(buffer.IsEmpty || buffer.Length != 0);
```

Comment:
But the test can run on zero length buffers. This test ensures that is buffer is not empty, then Length !=0